### PR TITLE
plugins: remove cmd/chainlink-starknet

### PR DESCRIPTION
**Pull request by jmank88**
https://smartcontract-it.atlassian.net/browse/BCF-2701

Requires:
- https://github.com/smartcontractkit/chainlink-starknet/pull/333

### CODE REVIEW

File: plugins/chainlink.Dockerfile
Comment (jmank88) [inline plugins/chainlink.Dockerfile:1]: `go install` would have been so much cleaner, but at least we know how to make this work already.

File: plugins/cmd/chainlink-starknet/README.md
Comment (jmank88) [inline plugins/cmd/chainlink-starknet/README.md:1]: @krehermann Where does this information belong?
I overlooked it at first, but now that I'm reading it, I'm not sure if it belongs in `chainlink-starknet`, or maybe `chainlink-relay` instead. WDYT?

Comment (patrickhuie19) [reply plugins/cmd/chainlink-starknet/README.md:1]: FWIW I vote `chainlink-starknet`

**Review by patrickhuie19 — APPROVED** [review]

File: plugins/cmd/chainlink-starknet/README.md
Comment (aalu1418) [reply plugins/cmd/chainlink-starknet/README.md:1]: +1 for `chainlink-starknet` since it's starknet specific

**Review by aalu1418 — APPROVED** [review]

**Comment by jmank88** [comment]
Updated with final starknet/relayer hash.

**Review by aalu1418 — APPROVED** [review]

**Review by patrickhuie19 — APPROVED** [review]

**Review by krehermann — APPROVED** [review]

File: plugins/cmd/chainlink-starknet/README.md
Comment (jmank88) [reply plugins/cmd/chainlink-starknet/README.md:1]: It is starknet specific but the keystore details leak through a bit in to the relayer.

**Pull request merged**